codebase/superdarn/src.lib/tk/fitacf_v3.0/src/determinations.c

```diff
@@ -511,16 +517,36 @@ void find_elevation(llist_node range, struct FitElv* fit_elev_array, FITPRMS* fi
     double real,imag;
     double psi_uncorrected_unfitted;
     double xcf0_p;
+ */ struct elevation_data *elev_data;
+    elev_data = malloc(sizeof(struct elevation_data));
+
+    if (elev_data == NULL || errno != 0)
+    {
+        fprintf(stderr,"Error: Malloc failed on generating elevation_data struct. %d",errno);
+    }
 
     RANGENODE* range_node;
 
     range_node = (RANGENODE*) range;
 
-
-    x = fit_prms->interfer_x;
-    y = fit_prms->interfer_y;
-    z = fit_prms->interfer_z;
-
+    elev_data->interfer_x = fit_prms->interfer_x;
+    elev_data->interfer_y = fit_prms->interfer_y;
+    elev_data->interfer_y = fit_prms->interfer_z;
```

Comment:
Not sure if this was fixed in your unpublished commits, but it should be `elev_data->interfer_z = fit_prms->interfer_z;`

Reply:
Hey, thanks for the catch sadly this is not the issue I have been dealing with. 
However, are we still wanting the ability to call the old elevation code? I noticed in RST 4.2 make_fit has no -old_elev option?